learn/wavemaker-release-notes/v10-5-6.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "WaveMaker 10.5.6 - Release date: 08 October 2020"
+id: ""
+sidebar_label: "Release v10.5.6"
+---
+*WaveMaker released a new update version with bug fixes.*
+
+## Improvements
+
+---
+
+- Stackable multiple toaster messages. This offers a way to display multiple toasters where you can configure the time duration.
+- Update of Android **targetSdkVersion** is required by November 2, 2020. For more information, see [Mobile Build - Manual](learn/hybrid-mobile/mobile-build-manual).
+
+## Bug Fixes
+
+---
+
+- Fixed issue related to exporting project shell to EDN.
+- Fixed issues related to editing and displaying the user profile name.
+- Fixed issue related to default messages which are not localized.
+- Fixed issue related to the **onSuccess** handler for file upload variable to customize the upload success notification.
+- Fixed issue related to the `reset()` method that was not working for an HTML upload widget.
```

Comment:
Change to, "On clicking reset on a form, the file widget inside it was not getting reset. This issue is now fixed"